tests/main/presenters/test_search_presenters.py

```diff
@@ -302,21 +315,26 @@ def test_filter_groups_have_correct_state_when_changed(self):
         })
 
         search_filters = list(filters_for_lot('paas', g6_builder).values())
-        set_filter_states(search_filters, request)
+        lots_by_slug = get_lots_by_slug(self._get_framework_fixture_data('g-cloud-6')['frameworks'])
+        clean_request_query_params = clean_request_args(request.args, search_filters, lots_by_slug)
```

Comment:
This test and the one above it share a fair amount of duplication so you could consider whether it can be parametrized (saying that it might impact a bit too much on readability so maybe have a think but don't do it if it's too bad for readability/ease)